cypress-io/cypress
# fix: change "resizable panels handlers" width to make it more convenient to use

Closes #22473
Closes https://github.com/cypress-io/cypress/issues/22608

### User facing changelog
Changed resizable panels' handlers width to make it more convenient to use

### Additional details
Resizable panels' handlers have too big width that overlap scrollbar.
As user, you have troubles with scrolling tests' log window.
That's why you need to do it less.
I resolved it with changing resizable panels' handlers' className: `w-16` -> `w-10`

### Steps to test
1. Open and run any test
2. Expend inforamtion in tests' log window to see a scrollbar (if it's not  visible)
3. Try to scroll using cursor

### How has the user experience changed?
Before: [video](https://recordit.co/jRUcJrPG0V)

After: [video](https://recordit.co/z384pudkL3)

### PR Tasks
- [na] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
